### PR TITLE
gh actions: release: fix the release flow

### PR DESCRIPTION
**Pull request by fromanirh**
Signed-off-by: Francesco Romani <fromani@redhat.com>

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ jobs:
     runs-on: ubuntu-20.04
     env:
       RTE_CONTAINER_IMAGE: quay.io/k8stopologyawareschedwg/resource-topology-exporter
+      E2E_TOPOLOGY_MANAGER_POLICY: SingleNUMANodeContainerLevel
```
Comment (Tal-or) [inline .github/workflows/release.yml:21]: Something is very suspicious, how the test passed if it is looking for:
https://github.com/k8stopologyawareschedwg/resource-topology-exporter/blob/88595fb0d865dedcf5a0333b977f1731d2ed0fa7/test/e2e/utils/nodetopology/nodetopology.go#L53

Comment (fromanirh) [reply .github/workflows/release.yml:21]: Good point, let's look into it a bit deeper

Comment (fromanirh) [reply .github/workflows/release.yml:21]: Previously, we were reading the expected value from the kubelet config file, so we required no env variable. This obviously works only on kind, we fixed this between 0.2.5 and now.

**Comment by fromanirh** [comment]
@Tal-or  any other comment on this PR?

**Comment by Tal-or** [comment]
> @Tal-or any other comment on this PR?

Nope. Just took a second look at the implementation.
/lgtm

**Pull request merged**